talk/objectorientation/advancedoo.tex

```diff
@@ -1,5 +1,44 @@
 \subsection[advOO]{Advanced OO}
 
+\begin{frame}[fragile]
+  \frametitlecpp[98]{\texttt{this} keyword}
+  \begin{block}{How to know an object's address?}
+    \begin{itemize}
+    \item Sometimes we need the address of the current object
+    \item Or we need to pass our address / a reference to a different entity
+      (for example to implement operators, see later)
+    \item All class methods can use the keyword \cppinline{this}
```

Comment:
Nit: Only non-static ones.

Reply:
True, but maybe we are awake enough to say this. This woulda allow us to explain what the difference between static and and non-static methods is.